src/Browsershot.php

```diff
@@ -29,6 +29,7 @@ class Browsershot
     protected $paperHeight = 0;
     protected $paperWidth = 0;
     protected $showBackground = false;
+    protected $omitScreenshotBackground = false;
```

Comment:
Rename this to `showScreenshotBackground` make it `true`by default. Change up the added code to make it work that way.

That'll be more readable.